alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportDeleteWorkflowAction.kt

```diff
@@ -296,17 +302,27 @@ class TransportDeleteWorkflowAction @Inject constructor(
             return deletableMonitors
         }
 
-        private suspend fun getWorkflow(): Workflow {
+        private suspend fun getWorkflow(): Optional<Workflow> {
```

Comment:
the workflow object is already nullable in kotlin. so, we do not need to use `java.util.Optional` here. https://typealias.com/guides/java-optionals-and-kotlin-nulls/
can we adjust this?